authorization-service/basicauthorizer.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+const dotenv = require("dotenv");
+dotenv.config();
+
+module.exports.basicAuthorizer = async (event, context, cb) => {
+  console.log(`Event: ${JSON.stringify(event)}`);
+
+  if (event.type !== "TOKEN") {
+    cb("Unauthorized");
+  }
+
+  try {
+    const authorizationToken = event.authorizationToken;
+
+    const encodedCreds = authorizationToken.split(" ")[1];
```

Comment:
I also suggest to check that `authorizationToken.split(" ")[0] === 'Basic'` and `!authorizationToken.split(" ")[1]` and throw `Unauthorized` if they are not valid.

Reply:
Another one good suggestion, thanks!